data/role/jenkins-dev.yaml

```diff
@@ -9,6 +9,13 @@ psick::jenkins::jcasc::options_hash:
     Name: 'jenkins-slave-windows-dev'
     Role: 'jenkins-slave'
     Env: 'dev'
+  linux_slave_ami: 'ami-f61ab08c'
```

Comment:
I prefer things to be ordered alphabetically within sections, but too late now, don't bother changing it. :)